packages/wallets/src/components/OptionsAndMultipliersListing/OptionsAndMultipliersListing.tsx

```diff
@@ -88,8 +88,16 @@ const OptionsAndMultipliersListing: React.FC<TOptionsAndMultipliersListingProps>
     const { data } = useActiveWalletAccount();
 
     useEffect(() => {
-        onOptionsAndMultipliersLoaded?.(true);
-        return () => onOptionsAndMultipliersLoaded?.(false);
+        let isMounted = true;
```

Comment:
Shouldn't isMounted be determined on `useEffect` with empty dependancy array?

Reply:
AFAIK isMounted could also be used on useEffects with dependencies. It is however best to use it in a useEffect with empty dependency to run the cleanup function only when component is unmounted. To avoid unexpected behaviour and potential issues I'll remove this for now as it is not a priority and out of scope 🙏